apps/i18n/dance/en_us.json

```diff
@@ -7,11 +7,13 @@
   "danceAiFeedbackCustomize": "⚠️ You need to change the effect A.I. generated. You can either regenerate the effects, customize the code A.I. created, or start over with new emojis.",
   "danceAiFeedbackEffectChangeAfterMeasures": "⚠️ Connect the `A.I. generate effect` block to the `after 2 Measures` event.",
   "danceAiFeedbackKeyPress": "⚠️ You pressed a key, but your dancer didn't do a new move.",
-  "danceAiFeedbackKeyPressRightOrLeft": "⚠️ Make sure to press at least one of the arrow keys while your dance is running.",
+  "danceAiFeedbackKeyPressRightOrLeft": "⚠️ Make sure to press the right or left arrow key while your dance is running.",
```

Comment:
Oh, do we say "arrow key" or just "arrow"?  Because we make the yellow arrow buttons available, and it seems likely a lot of students might find them more obvious.